### PR TITLE
Create compaxion.1tb.uk.js

### PR DESCRIPTION
**Pull request by max8330**
### Link to Website

Link: 

---

- [ ] Toggle this checkbox to re-run the stargazer checking

### CODE REVIEW

**Comment by tarampampam** [comment]
Fix the errors, please. And format the file

**Review by tarampampam — CHANGES_REQUESTED** [review]
Changes required

**Comment by max8330** [comment]
Failed

**Review by max8330 — COMMENTED** [review]
Have the errors corrected, thanks.

**Comment by tarampampam** [comment]
Fix the file name, please (1bt instead of 1tb)

File: domains/compaxion.1tb.uk.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    repo: 'https://github.com/smax8330/free-domains/',
+    email: 'mosynikrooy@gmail.com',
+  },
+  record: {
+    NS: [
+      'ns1.digitalocean.com.',
+      'ns2.digitalocean.com.',
+      'ns3.digitalocean.com.',
+      
+
```
Comment (tarampampam) [inline domains/compaxion.1tb.uk.js:15]:
```suggestion
```

Remove empty lines, please

**Review by tarampampam — CHANGES_REQUESTED** [review]

**Comment by max8330** [comment]
Fixed the filename, thanks

**Comment by max8330** [comment]
Unused spaces were removed, thanks.

**Review by tarampampam — APPROVED** [review]

**Comment by tarampampam** [comment]
A record cannot be used together with NS. So, I've dropped the A record, ok?

**Comment by max8330** [comment]
But in order to link the domain to my digital Ocean server it must be set
for both A and NS to work perfectly.

Or am I wrong in this matter?

**Comment by tarampampam** [comment]
I think not. Just one of those types is required (`NS` delegates the resolution to another server and `A` forwards requests directly to your origin server). So, you need to select something one

**Comment by max8330** [comment]
Then I think I'm ok with that, which one is the best record needed? A or NS?

**Comment by max8330** [comment]
I think A record works best for me, please drop the NS record and set only
A record.


Thanks.

**Comment by tarampampam** [comment]
Done in https://github.com/tarampampam/free-domains/commit/1e453cab46166b06507d7661ae8bd49639dec9d9

**Pull request merged**